src/main/resources/application.properties

```diff
@@ -23,6 +23,7 @@ detect.bom.aggregate.name=
 detect.excluded.bom.tool.types=
 detect.included.bom.tool.types=
 detect.suppress.configuration.output=
+detect.api.timeout=
 detect.suppress.results.output=
 
```

Comment:
please remove the old policy check timeout that the detect.api.timeout replaces

Reply:
Missed that one, now removed